cmd/eval.go

```diff
@@ -147,6 +161,9 @@ Set the output format with the --format flag.
 	evalCommand.Flags().VarP(params.explain, "explain", "", "enable query explainations")
 	evalCommand.Flags().VarP(params.outputFormat, "format", "f", "set output format")
 	evalCommand.Flags().BoolVarP(&params.profile, "profile", "", false, "perform expression profiling")
+	evalCommand.Flags().VarP(&params.profileCriteria, "profile-sort", "", "criteria to sort the expression profiling results")
```

Comment:
Nit: `"set sort order of expression profiler results"`  to be consistent with other setter options.

Reply:
Done